app/static/recipe.js

```diff
@@ -0,0 +1,73 @@
+function handleErrors(response) {
+    if (!response.ok) {
+        throw Error(response.statusText);
+    }
+    return response.json();
+}
+
+function get(url, callback, error) {
+    fetch(url)
+        .then(handleErrors)
+        .then(callback)
+        .catch(error);
+}
+
+// Builds the HTML Table out of myList.
+function buildHtmlTable(output, response) {
+    // Header - item 0
+    var header = $('<tr/>');
+    var headerData = response.shift();
+    headerData.forEach(function(headerCell){
+        header.append($('<th/>').html(headerCell));
+    });
+    output.append(header);
+
+    // Data - the rest
+    response.forEach(function(rowData) {
+        var row = $('<tr/>');
+        rowData.forEach(function(cellData){
+            row.append($('<td/>').html(cellData));
+        });
+        $(output).append(row);
+    });
+}
+
+function run_recipe(url, params) {
```

Comment:
nit: camelCase is preferred for JS (and you use it in other places in this file, so better to be consistent anyway).

Reply:
Thank you I updated :-)